### DISCUSSION TITLE
Gettint error "You don't have a network connection"

### DISCUSSION THREAD

**Discussion by KottaKritter** [LSPosed/MagiskOnWSALocal#320] [Q&A] [answered]
I have installed MagiskonWSA with Google Apps, but when I click on "Sign in" I get the error "You don't have a network connection" I already verified that my firewall is allowing WSA to connect to private and public networks. Is there anything I'm missing?

**Comment by Howard20181** [comment, marked as answer]
Install a browser in WSA to test whether the network is available.
If you cannot access Google services in your area, you also need to set up a proxy in the WSA, which is not interchangeable with the proxy settings of Windows.
